pkgs/build-support/builder-defs/builder-defs.nix

```diff
@@ -569,7 +569,7 @@ let inherit (builtins) head tail trace; in
      # Interpreters that are already in the store are left untouched.
          echo "patching script interpreter paths"
          local f
-         for f in $(find "${dir}" -xtype f -perm /0100); do
+         for f in $(find "${dir}" -xtype f ${findExecutablesFlags:--perm /0100}); do
```

Comment:
The `${` here (when used as the shell construct) needs to be escaped in .nix files like this: `''${`, or it'll do something unexpected (try e.g. `nix-build -A fbpanel` with this).